packages/ui-components/src/components/toolbar.tsx

```diff
@@ -870,6 +886,9 @@ export function ToolbarButtonComponent(
       aria-disabled={disabled}
       aria-label={props.label || title}
       aria-pressed={props.pressed}
+      aria-haspopup={ariaHaspopup}
```

Comment:
Would this pattern work?

```suggestion
      aria-haspopup={props['aria-haspopup']}
```

It would avoid defining intermediary variables.